### DISCUSSION TITLE
Test alias

### DISCUSSION THREAD

**Discussion by gatewaycat** [dbt-labs/dbt-core#5664] [Ideas]
Allow an alias config for tests declared in yml files. Print the alias instead of the autogenerated test name when one runs `dbt test`. Also name the SQL file generated for the test in the target folder with this alias.

**Comment by jtcohen6** [comment]
Hey @gatewaycat, check out: https://docs.getdbt.com/reference/resource-properties/tests#define-a-custom-name-for-one-test

Starting in v1.1, we made it possible to define a custom `name` for a specific instance of a generic test. It's _also_ possible to configure a custom `alias`, which will take effect if you're materializing your test in the data warehouse by means of [`--store-failures`](https://docs.getdbt.com/reference/resource-configs/store_failures).
